timed/tracking/serializers.py

```diff
@@ -39,12 +39,20 @@ def validate(self, data):
         to_time = data.get('to_time', instance and instance.to_time)
         user = instance and instance.user or data['user']
 
+        def validate_unique():
```

Comment:
Sounds quite generic and could be anything. Maybe `validate_running_activity` might be a better name? Or any other ideas?

Reply:
Couldn't think of a better name, so i just used your suggestion